dev-cpp/yaml_cpp/yaml_cpp-0.6.1.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A YAML parser and emitter in C++"
+DESCRIPTION="yaml-cpp is a YAML parser and emitter in C++ matching the YAML \
+1.2 spec."
+HOMEPAGE="https://github.com/jbeder/yaml-cpp"
+COPYRIGHT="2018 Jesse Beder"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-$portVersion.tar.gz"
+CHECKSUM_SHA256="25ec37e6d82ab8c485926d69a5567741c7263515f8631e5dcb3fb4708e6b0d0d"
+SOURCE_DIR="yaml-cpp-yaml-cpp-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	yaml_cpp$secondaryArchSuffix = $portVersion compat >= 0.6
+	lib:libyaml_cpp$secondaryArchSuffix = $portVersion compat >= 0.6
```

Comment:
How about using `libVersionCompat`? You should also drop compat version for `yaml_cpp`.

Looks like the devs [forgot][0] to bump soname to 0.6.1, or intentionally not doing so. Either way, we should keep `libVersion` at 0.6.0, as it's the soname version.

```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="0.6.0"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+
 PROVIDES="
-	yaml_cpp$secondaryArchSuffix = $portVersion compat >= 0.6
-	lib:libyaml_cpp$secondaryArchSuffix = $portVersion compat >= 0.6
+	yaml_cpp$secondaryArchSuffix = $portVersion
+	lib:libyaml_cpp$secondaryArchSuffix = $libVersionCompat
```

[0]: https://github.com/jbeder/yaml-cpp/blob/5d5bb52ec2825eb4c0c932a9c2d50dc3699f932f/CMakeLists.txt#L32